### PR TITLE
Add additional evidence/assertion fields to the CIViC consequence annotation

### LINKED ISSUE #88: Add more fields to the VCF annotation output
Right now we only support basic assertion and evidence information. We should add at minimum all fields shared between assertions and evidence items.

### PR DESCRIPTION
**Pull request by susannasiebert**
Closes #88

### CODE REVIEW

File: civicpy/civic.py [hunk 1]
```diff
@@ -638,6 +638,19 @@ def csq(self, include_status=None):
                     "{} ({})".format(evidence.source.citation_id, evidence.source.source_type),
                     str(evidence.variant_origin),
                     evidence.status,
+                    str(evidence.clinical_significance or ''),
+                    str(evidence.evidence_direction or ''),
+                    "{} (DOID {})".format(evidence.disease.name, evidence.disease.doid),
```
Comment (ahwagner) [inline civicpy/civic.py:643]: Not sure about this–do all CIViC diseases correspond to DO terms? If there is no doid, do we want to display as "<Disease> (DOID )"?

File: civicpy/civic.py [hunk 2]
```diff
@@ -638,6 +638,19 @@ def csq(self, include_status=None):
...
+                    '&'.join(["{} (NCIt ID {})".format(drug.name, drug.ncit_id) for drug in evidence.drugs]),
```
Comment (ahwagner) [inline civicpy/civic.py:644]: Same comment regarding drugs as above–do all drugs have NCIt IDs?

File: civicpy/civic.py [hunk 3]
```diff
@@ -638,6 +638,19 @@ def csq(self, include_status=None):
...
+                    str(evidence.drug_interaction_type or ""),
+                    '&'.join(["{} (HPO ID {})".format(phenotype.hpo_class, phenotype.hpo_id) for phenotype in evidence.phenotypes]),
```
Comment (ahwagner) [inline civicpy/civic.py:646]: Same as above 2 comments–do all phenotypes have HPO IDs?

File: civicpy/civic.py [hunk 4]
```diff
@@ -664,6 +677,19 @@ def csq(self, include_status=None):
                     "",
                     str(assertion.variant_origin),
                     assertion.status,
+                    assertion.clinical_significance,
+                    assertion.evidence_direction,
+                    "{} (DOID {})".format(assertion.disease.name, assertion.disease.doid),
+                    '&'.join(["{} (NCIt ID {})".format(drug.name, drug.ncit_id) for drug in assertion.drugs]),
```
Comment (ahwagner) [inline civicpy/civic.py:683]: See above comments re: ID provided

File: civicpy/civic.py (hunk 3 above)
Comment (susannasiebert) [reply civicpy/civic.py:646]: Yes, all phenotypes have a HPO ID. The same is not true for disease and drugs

File: civicpy/civic.py (hunk 1 above)
Comment (susannasiebert) [reply civicpy/civic.py]: Resolved by overwriting the `Disease` class `__str__` method.

File: civicpy/civic.py (hunk 2 above)
Comment (susannasiebert) [reply civicpy/civic.py]: Resolved by overwriting the `Drug` class `__str__` method.

**Pull request merged**